srcts/src/bindings/input/selectInput.ts

```diff
@@ -292,8 +294,22 @@ class SelectInputBinding extends InputBinding {
       control.destroy();
       control = $el.selectize(settings)[0].selectize as SelectizeInfo;
     }
+
+    // (Hopefully temporary) workaround for a v0.15.2 selectize bug where the
+    // dropdown (instead of the <input>) gets focus after an item added via
+    // click. https://github.com/selectize/selectize.js/issues/2032
```

Comment:
The most minor of comments, but I use extensions that find `FIXME` and `TODO` comments that make it easier to find these things again later.

```suggestion
    // (Hopefully temporary) workaround for a v0.15.2 selectize bug where the
    // dropdown (instead of the <input>) gets focus after an item added via
    // click. 
    // FIXME: https://github.com/selectize/selectize.js/issues/2032
```